docs/components/zeebe/technical-concepts/internal-processing.md

```diff
@@ -64,6 +64,8 @@ When the broker receives more requests than it can process with an acceptable la
 
 Backpressure is indicated to the client by throwing a **resource exhausted** exception. If a client sees this exception, it can retry the requests with an appropriate retry strategy. If the rejection rate is high, it indicates the broker is constantly under high load and you need to reduce the rate of requests. Alternatively, you can also increase broker resources to adjust to your needs. In high-load scenarios, it is recommended to [benchmark](https://camunda.com/blog/2022/05/how-to-benchmark-your-camunda-platform-8-cluster/) your Zeebe broker up front to size it correctly.
 
+<!--- Would it be worth adding documentation for benchmarking? --->
```

Comment:
This is the question of the moment. On our [Contact page](https://docs.camunda.io/contact/) we ask that people reach out about performance and benchmarking. I _think_ this is so we can hand hold folks through the process and ensure they get the correct setup and outcome. 

My guess is this is a PM or even consulting question. You could also follow up with support if you want to get more data on whether customers are asking for documentation on benchmarking or just asking for assistance in benchmarking.

Reply:
Inquired on Slack about this 👍